Readme.md

```diff
@@ -53,4 +53,4 @@ Checkout Docker Installation [docs](./docs/Docker-installation.md)
 ## Get involved!
 
 * Start by reading our [Code of conduct](https://github.com/hotosm/fAIr/blob/master/docs/Code-of-Conduct.md)
-* Get familiar with our [contributor guidelines](https://github.com/hotosm/fAIr/blob/master/docs/Contribution.md) explaining the different ways in which you can support this project! We need your help!
+* Get familiar with our [contributor guidelines](https://github.com/hotosm/fAIr/blob/master/CONTRIBUTING.md) explaining the different ways in which you can support this project! We need your help!
```

Comment:
Use relative file path , So that it will work on any branch and fork

Reply:
okay @kshitijrajsharma on it now. Will let you know when its done